### PR TITLE
PS-1553 Do not lock Keboola dependecies on minor versions

### PR DESCRIPTION
**Pull request by joker806**
https://keboola.atlassian.net/browse/PS-1553

This is needed to be able to update dependencies in `keboola/data-loader-api`

### CODE REVIEW

**Comment by joker806** [comment]
Sry, too soon.. jeste musim zupdatovat `storage-api-php-client-branch-wrapper`

File: composer.json [hunk 1]
```diff
@@ -20,23 +20,22 @@
...
-        "symfony/finder": "^2.8.52|^4.1",
-        "symfony/serializer": "^2.8.52|^4.1",
-        "monolog/monolog": "^1.25.5|^2.0",
-        "keboola/input-mapping": "^12.0.0",
-        "keboola/storage-api-php-client-branch-wrapper": "^1.0.0",
+        "ext-json": "*",
         "guzzlehttp/guzzle": "^6.5.5",
+        "keboola/input-mapping": "^12.0",
         "keboola/sanitizer": "^0.1",
+        "keboola/storage-api-php-client-branch-wrapper": "^1.0.0",
```
Comment (pivnicek) [inline composer.json:27]: maybe we also want this as "^1.0" rather?

Comment (joker806) [reply composer.json:27]: yes, that's what I meant by the last comment :+1:

**Comment by odinuv** [comment]
Toto tu nějak hnije

**Comment by joker806** [comment]
Hele, evidentne to nebylo potreba :man_facepalming: nebo se to vyresilo s neicm jinym. Zahazuju

**Pull request closed without merging**